test/snapshot/snapshot.go

```diff
@@ -0,0 +1,113 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package snapshot
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"math/rand"
+	"os"
+	"strings"
+
+	"github.com/cartesi/rollups-node/pkg/emulator"
+)
+
+const (
+	ramLength = 64 << 20
+)
+
+var ImagesPath = "/usr/share/cartesi-machine/images/"
```

Comment:
This is treated as a constant in the code

Reply:
It is global that can be reset if the images are located in another folder.
I've added documentation to explain that.